nubilanding/src/components/beneficios/styles.module.scss

```
@@ -0,0 +1,65 @@
+@import url("https://fonts.googleapis.com/css?family=Roboto:400,500|Rubik:300,400,500,500i&display=swap");
+
+.contenedorGeneral {
+  width: 100%;
+  height: 790px;  /*lo agrande para igualar la altura  entre los primeros parrafos(top) y el boton(botton)*/  
+  background-color: #eeeeef;
+}
+
+.contenedorBeneficios {
+  width: 100%;  /*remember no tiene que tener margin para agrandar el top*/
+  height: 50px;
+  display: flex;  
+  flex-wrap: wrap;
+  justify-content: center;
+  border-top: transparent solid 70px; /*para agrandar el top de los primeros parrafos*/
+}
+
+.contenedorParrafos {
+  width: 30%; /*el ancho esta bien en web*/
+  height: 250%;
+  text-align: justify;
+  padding: 15px;
+  margin: 50px 50px 30px 25px;
```

Quisas quedaria mejor que tengan un margen todos parejos, ir probando hasta que te qeden bien acomodados. Por ejemplo margin : 40 px.
```suggestion
  margin: 40px;
```